crates/lib/Cargo.toml

```
@@ -0,0 +1,75 @@
+[package]
+name = "binstall"
+description = "Library backend for cargo-binstall"
+repository = "https://github.com/cargo-bins/cargo-binstall"
+documentation = "https://docs.rs/binstall"
+version = "0.12.0"
+rust-version = "1.61.0"
+authors = ["ryan <ryan@kurte.nz>"]
+edition = "2021"
+license = "GPL-3.0"
+
+[dependencies]
+async-trait = "0.1.57"
+bytes = "1.2.1"
+bzip2 = "0.4.3"
+cargo_toml = "0.11.5"
+clap = { version = "3.2.16", features = ["derive"] }
+compact_str = { version = "0.5.2", features = ["serde"] }
+crates_io_api = { version = "0.8.0", default-features = false }
+dirs = "4.0.0"
+flate2 = { version = "1.0.24", default-features = false }
+fs4 = "0.6.2"
+futures-util = { version = "0.3.21", default-features = false }
+home = "0.5.3"
+itertools = "0.10.3"
+jobserver = "0.1.24"
+log = "0.4.17"
```

We actually need to enable the `std` feature of log here:

```suggestion
log = { version = "0.4.17", features = "std" }
```

We didn't do that because `simplelog` automatically enable that feature.

---

Oops, I forgot the `[]`:

```suggestion
log = { version = "0.4.17", features = ["std"] }
```